src/Rules/CheckTypehintCallerTypeRule.php

```diff
@@ -143,16 +145,7 @@ private function validateArgVsParamTypes(array $args, MethodCall $methodCall, Sc
                 continue;
             }
 
-            $argType = $scope->getType($arg->value);
```

Comment:
moved into `validateParam` so we don't need to resolve the type for non-fully-qualified types